src/server.cpp

```diff
@@ -3669,7 +3669,7 @@ void Server::requestShutdown(const std::string &msg, bool reconnect, float delay
 	// Positive delay, tell the clients when the server will shut down
 		std::wstringstream ws;
 
-		ws << L"*** Server shutting down in "
+		ws << countdown
 				<< duration_to_string(myround(delay)).c_str()
 				<< ".";
```

Comment:
It would be useful if this could be formatted instead of being hardcoded as "Text <seconds>."

Reply:
done.